python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -239,13 +240,18 @@ def groups(*group_strs, include_sources: bool = False) -> "GroupsAssetSelection"
             include_sources (bool): If True, then include external assets matching the group in the
                 selection.
         """
-        check.tuple_param(group_strs, "group_strs", of_type=str)
-        return GroupsAssetSelection(selected_groups=group_strs, include_sources=include_sources)
+        check.tuple_param(group_strs, "group_strs", of_type=Optional[str])
+        return GroupsAssetSelection(
+            selected_groups=list(filter(lambda x: x is not None, group_strs)),
```

Comment:
this is the same as `list(filter(None, group_strs))`, so could be a bit cleaner

separate from that, it looks like if you pass in None, then we just ignore that which seems like not the behavior we want -- ideally we would just directly pass the group_strs through here

Reply:
`list(filter(None, group_strs))` was filtering out empty strings. But yeah you're right I can just do `group_strs=[]`. Sorry my python is rusty